examples/blkdiscard/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/metal-toolbox/ironlib/utils"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	device  = "/dev/sdZZZ"
+	timeout = 2 * time.Minute
+)
+
+// This example invokes ironlib and runs blkdiscard on the disk /dev/sdZZZ
```

Comment:
can you add an arg so this example is a little more useful? I did that for nvme-wipe and actually found some errors which was nice :D https://github.com/metal-toolbox/ironlib/blob/main/examples/nvme-wipe/main.go#L13

Reply:
thanks!